galaxy/api/internal/views/combined.py

```diff
@@ -48,11 +48,11 @@ def get(self, request):
                 detail='The namespace parameter is required')
 
         collection_filters = {
-            'namespace__name': namespace,
+            'namespace__name__iexact': namespace,
```

Comment:
Do we need this change since we _are_ enforcing collections only in lowercase-compliant namespaces?